### ISSUE TITLE
does not compile with nasm of centos-7.4

### ISSUE THREAD

**Issue by holgerBerger** [travisdowns/avx-turbo#2]
```
nasm -DNASM_ENABLE_DEBUG=0 -w+all -l x86_methods.list -f elf64 asm-methods.asm
asm-methods.asm:96: error: parser: instruction expected
asm-methods.asm:97: error: parser: instruction expected
asm-methods.asm:98: error: symbol `test_func_sparse' redefined
asm-methods.asm:98: error: parser: instruction expected
asm-methods.asm:99: error: symbol `test_func_sparse' redefined
asm-methods.asm:99: error: parser: instruction expected
```

``` 
nasm -v
NASM version 2.10.07 compiled on Jun  9 2014
```

**Comment by travisdowns** [comment]
@holgerBerger - right, it needs a newer nasm version that understands AVX-512. I will bundle one like I did [over here](https://github.com/travisdowns/uarch-bench/blob/master/nasm-2.13.03/NOTE) to make things "just work". Working on it now.

**Comment by holgerBerger** [comment]
Got I running with a file generated on Ubuntu. See the results in another
ticket.

travisdowns <notifications@github.com> schrieb am Fr., 24. Aug. 2018, 23:15:

**Comment by travisdowns** [comment]
@holgerBerger - thanks, I will look in a moment.

I fixed the nasm issue [over here](https://github.com/travisdowns/avx-turbo/commit/4b5d6bc74b01eba13211c08b96927e9342c744f1) and I'll merge into master shortly (getting TravisCI up and running now).

**Comment by travisdowns** [comment]
Fixed in 4b5d6bc74b01eba13211c08b96927e9342c744f1.
